packages/blade/src/components/Checkbox/Checkbox.tsx

```diff
@@ -241,13 +241,18 @@ const _Checkbox: React.ForwardRefRenderFunction<BladeElementRef, CheckboxProps>
               isNegative={_hasError}
             />
             {children ? (
-              <SelectorTitle size={_size} isDisabled={_isDisabled}>
-                {children}
-              </SelectorTitle>
+              <BaseBox onClick={(e) => e.stopPropagation()}>
+                <SelectorTitle size={_size} isDisabled={_isDisabled}>
```

Comment:
Is it possible to have onClick on SelectorTitle itself instead of introducing a BaseBox wrapper?